tests/test_PETSc.py

```diff
@@ -113,19 +115,26 @@ def test_norm():
     nx = 4
     ny = nx
     nz = nx
+    dim = 3
     dof = 4
     n = nx * ny * nz * dof
+    parameters = {}
 
     state = numpy.zeros(n)
     for i in range(n):
         state[i] = i + 1
 
-    state_dist = PETScInterface.Vector.from_array(state)
+    interface = PETScInterface.Interface(parameters, nx, ny, nz, dim, dof)
+
+    state_dist = PETScInterface.Vector.from_array(interface.map, state)
```

Comment:
In the Epetra interface I have a method that does it so you don't need to know about the map as a user: https://github.com/BIMAU/transiflow/blob/master/transiflow/interface/Epetra.py#L88